src/assets/styles/components/_card.css

```diff
@@ -42,6 +42,7 @@ body:not([class*="fl-theme"]),
 }
 
 .card__image {
+    border-top: 0.125rem solid var(--fl-fgColor, var(--color-black)); 
     min-height: 14.44rem;
     min-width: 20.5rem;
```

Comment:
Can you remove the min-height? It's preventing the image from completely covering the card here:

<img width="376" alt="Screenshot 2025-04-23 at 4 09 45 PM" src="https://github.com/user-attachments/assets/c8468a4d-c612-420e-954a-4d1d7cbb74cd" />